### PR TITLE
JBIDE-25870 - Add CDK runtime download integration tests

### PR DESCRIPTION
**Pull request by odockal**
- ITests for downloading container runtimes CDK 3.x, CDK 3.2+ and Minishift 1.14
   - Checking passing of credentials in download runtime wizard
   - Tests downloading into default folders

Signed-off-by: Ondrej Dockal <odockal@redhat.com>

# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by odockal** [comment]
testPR

**Comment by odockal** [comment]
Waiting for jenkins build to actually run a job with tests. Lot of problems recently.

**Comment by odockal** [comment]
Rhel7 and win10 green jobs: https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Devstudio/view/devstudio_integration_tests/job/cdk.itests/590/, there are some fails on win7, though...

**Comment by odockal** [comment]
All green: https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Devstudio/view/devstudio_integration_tests/job/cdk.itests/592/

**Comment by odockal** [comment]
@jkopriva  could you take a look, please?

File: itests/org.jboss.tools.cdk.ui.bot.test/src/org/jboss/tools/cdk/ui/bot/test/CDKAllTestsSuite.java [hunk 1]
```diff
@@ -34,10 +39,17 @@
 	// UI integration tests
 	
 	// Wizard tests by server adapter type
-	CDK3ServerWizardTest.class,
+	/*CDK3ServerWizardTest.class,
```
Comment (jkopriva) [inline itests/org.jboss.tools.cdk.ui.bot.test/src/org/jboss/tools/cdk/ui/bot/test/CDKAllTestsSuite.java]: @odockal what is the purpose of this have commented?

File: test-framework/org.jboss.tools.cdk.reddeer/src/org/jboss/tools/cdk/reddeer/core/enums/CDKRuntimeOS.java [hunk 2]
```diff
@@ -12,37 +12,44 @@
...
+ * 
  * @author odockal
  *
  */
 public enum CDKRuntimeOS {
 
-	LINUX("linux", ""),
-	WINDOWS("win", ".exe"),
-	MAC("darwin", ""),
+	LINUX("linux", "linux", ""),
```
Comment (jkopriva) [inline test-framework/org.jboss.tools.cdk.reddeer/src/org/jboss/tools/cdk/reddeer/core/enums/CDKRuntimeOS.java]: @odockal maybe there could comment what these strings means...

Comment (odockal) [reply test-framework/org.jboss.tools.cdk.reddeer/src/org/jboss/tools/cdk/reddeer/core/enums/CDKRuntimeOS.java]: updated in class javadoc.

**Comment by jkopriva** [comment]
testPR

**Comment by jkopriva** [comment]
LGTM, merging :)

**Pull request merged**